configs/mount.go

```diff
@@ -18,4 +18,10 @@ type Mount struct {
 
 	// Relabel source if set, "z" indicates shared, "Z" indicates unshared.
 	Relabel string `json:"relabel"`
+
+	// Optional Command to be run before Source is mounted.
+	PremountCmds [][]string `json:"premount_cmds"`
```

Comment:
Maybe we can use some structure which looks like `exec.Cmd`? I think Env and Dir parameters can be useful.